Content.Shared/Chat/TypingIndicator/TypingIndicatorComponent.cs

```diff
@@ -16,5 +16,9 @@ public sealed partial class TypingIndicatorComponent : Component
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite)]
     [DataField("proto", customTypeSerializer: typeof(PrototypeIdSerializer<TypingIndicatorPrototype>))]
-    public string Prototype = SharedTypingIndicatorSystem.InitialIndicatorId;
+    public string Prototype = "default";
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("overrideIndicator", customTypeSerializer: typeof(PrototypeIdSerializer<TypingIndicatorPrototype>))]
+    public string? OverrideIndicator = null;
```

Comment:
replace this with
```cs
[DataField]
public ProtoId<TypingIndicatorPrototype> OverrideIndicator;
```

Reply:
Should I replace the other one as well?

Reply:
might as well

Reply:
Yeah also document this one.

Reply:
OK did it! I'm turning this to draft though because I want to change some of the variable names (It being named Prototype doesn't make sense) and I don't have time right now.